### PR TITLE
setup/android_build_env: Small update

### PR DESCRIPTION
**Pull request by arizaini**
* Remove duplicate package and sort it by name
* Add curl package for Ubuntu 18
* Use openjdk-11-* instead

Signed-off-by: arizaini <famegalaxy614@gmail.com>

### CODE REVIEW

File: setup/android_build_env.sh [hunk 1]
```diff
@@ -7,26 +7,30 @@
...
-libssl-dev clang cmake axel re2c "${PACKAGES}"
+sudo apt install -y adb autoconf automake axel bc bison build-essential clang cmake expat fastboot flex \
+g++ g++-multilib gawk gcc gcc-multilib gnupg gperf htop imagemagick lib32ncurses5-dev lib32z1-dev \
+libc6-dev libcap-dev libcloog-isl-dev libexpat1-dev libgmp-dev liblz4-* liblzma* libmpc-dev libmpfr-dev \
+libncurses5-dev libsdl1.2-dev libssl-dev libtool libxml2 libxml2-utils lzma* lzop maven ncftp ncurses-dev \
+openjdk-8-jdk openjdk-8-jre patch pkg-config pngcrush pngquant python python-all-dev re2c schedtool \
+squashfs-tools subversion texinfo unzip w3m xsltproc zip zlib1g-dev "${PACKAGES}"
 # Purge problematic packages found in things like Mint 19
-sudo apt purge -y openjdk-11-jre openjdk-11
+sudo apt purge -y openjdk-11-*
```
Comment (MSF-Jarvis) [inline setup/android_build_env.sh]: Why this?

Comment (arizaini) [reply setup/android_build_env.sh]: this will remove jdk 11 related and it's dependency

Comment (MSF-Jarvis) [reply setup/android_build_env.sh]: Huh. Do seem to remember additional leftovers so guess we'll roll with this.

**Comment by MSF-Jarvis** [comment]
Thanks for this!

File: setup/android_build_env.sh (hunk 1 above)
Comment (arizaini) [reply setup/android_build_env.sh]: openjdk-11(headless actually) is automatically installed while running android_build_env.sh so need to be removed, i'm running ubuntu 18 btw

**Pull request merged**